src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -343,13 +343,17 @@ public AuthUrlBuilder setApprovalPrompt(boolean approvalPrompt) {
       return this;
     }
 
+    public AuthUrlBuilder setMakeBypass(String make) {
+      urlBuilder.addQueryParameter("make", make);
+      return this;
+    }
+
     public AuthUrlBuilder setSingleSelect(boolean singleSelect) {
       urlBuilder.addQueryParameter("single_select", Boolean.toString(singleSelect));
       return this;
     }
-
-    public AuthUrlBuilder setMakeBypass(String make) {
-      urlBuilder.addQueryParameter("make", make);
+    public AuthUrlBuilder setSingleSelectVin(String vin) {
+      urlBuilder.addQueryParameter("single_select_vin", vin);
```

Comment:
Same question as I asked on Android. Do we want the developer to have to explicitly `.setSingleSelect(true)`? As long as this is intentional, that's cool with me! Just want to make sure.

Reply:
intentional!